app/src/main/java/com/example/busschedule/ui/BusScheduleViewModel.kt

```diff
@@ -23,7 +23,7 @@ import com.example.busschedule.data.BusSchedule
 import kotlinx.coroutines.flow.Flow
 import kotlinx.coroutines.flow.flowOf
 
-class BusScheduleViewModel(): ViewModel() {
+class BusScheduleViewModel: ViewModel() {
```

Comment:
Per the [style guide](https://developer.android.com/kotlin/style-guide), there should be one space before the ":".